src/test/java/ezschedule/testutil/EventBuilder.java

```diff
@@ -29,6 +33,15 @@ public EventBuilder() {
         startTime = new Time(DEFAULT_START_TIME);
         endTime = new Time(DEFAULT_END_TIME);
     }
+    /**
+     * Creates a {@code EventBuilder} with the given details.
+     */
+    public EventBuilder(String name, String date, String startTime, String endTime) {
+        this.name = new Name(name);
+        this.date = new Date(date);
+        this.startTime = new Time(startTime);
+        this.endTime = new Time(endTime);
+    }
```

Comment:
Not necessary. No need to add more code to maintain.

The same thing can be achieved by calling
``` java
Event e = new EventBuilder()
                  .withName(name)
                  .withDate(date)
                  .withStartTime(startTime)
                  .withEndTime(endTime)
                  .build();
```